adoc/bp_postgresql_database_migration.adoc

```diff
@@ -96,12 +99,106 @@ Only perform a fast migration if you have an availabel database backup.
 ====
 
 
-Perform a fast migration with the following command (Ensure you have a database backup):
+Perform a fast migration with the following command (ensure you have a database backup):
 
 ----
 $> /usr/lib/susemanager/bin/pg-migrate.sh fast
 ----
 
+
+
+=== PostgreSQL 9.6 to 10
+
+==== Prepare to Upgrade
+
+Before you begin the upgrade, prepare your existing 3.2 system and create a database backup.
+
+PostgreSQL stores data at [path]``/var/lib/pgsql/data/``, and logs to [path]``/var/lib/pgsql/data/pg_xlog/``.
+
+.Procedure: Preparing to Upgrade
+
+. Check the active PostgreSQL version:
++
+----
+psql --version
+----
++
+If you are using PostgreSQL{nbsp}9.6, you can upgrade to PostgreSQL{nbsp}10.
```

Comment:
Check the {nbsp} throughout, I can't tell if you've put them in or not.

Reply:
yes, might be worth checking